File: core/android/app/src/main/java/com/aco/fitar/api/models/RegistrationModel.kt
package com.aco.fitar.api.models

import com.aco.fitar.api.ModelBase
import com.aco.fitar.api.ModelResponseBase

class RegistrationModel(var username:String, var firstName:String, var lastName:String, var password:String) : ModelBase() {

}

class RegistrationResponseModel: ModelResponseBase(){}

File: core/FitAR/FitAR.API/Models/RegistrationModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace FitAR.Web.API.Models
{

  public class RegistrationInputModel
  {
    public string username { get; set; }
    public string firstName { get; set; }
    public string lastName { get; set; }
    public string password { get; set; }
  }

  public class RegistrationModel : ModelBase
  {

  }
}
